a3p-integration/proposals/c:stake-bld/wallet.js

```diff
@@ -0,0 +1,260 @@
+// @ts-check
```

Comment:
this file is mostly copied from `agoric-cli`. I think we'll want to DRY it but I don't know when. I'm reluctant to have more exports from agoric-cli. I could see them going in `ui-kit` though. Or `@agoric/contracts` as support tools. I figure it's okay to duplicate for tests .